lib/components/viewers/live-stop-times.tsx

```diff
@@ -151,25 +176,55 @@ class LiveStopTimes extends Component<Props, State> {
       return stopTimeComparator(stopTimesA[0], stopTimesB[0])
     }
 
+    const configDepartureWindow = stopViewerConfig?.timeRange / 86400
+
     const refreshButtonText = intl.formatMessage({
       id: 'components.LiveStopTimes.refresh'
     })
 
+    const routeTimes = Object.values(stopTimesByPattern)
+      .filter(
+        ({ pattern, route, times }) =>
+          times &&
+          times.length !== 0 &&
+          routeIsValid(route, getRouteIdForPattern(pattern))
+      )
+      .sort(patternComparator)
+      .map((route) => {
+        const sortedTimes = route.times
+          .concat()
+          ?.sort(stopTimeComparator)
+          // Only show times within the next 2 days, unless otherwise indicated by the config.
+          .filter((time: any, i: number, array: Array<any>) => {
+            const departureWindow = configDepartureWindow || 4
```

Comment:
Isn't this variable redundant now? (`configDepartureWindow` always defined -- assuming it is probably non-zero?)